tests/test_emission_manager.cairo

```diff
@@ -7,26 +7,74 @@ from contracts.interfaces.i_emission_manager import IEmissionManager
 const OWNER = 111
 const REWARDS_CONTROLLER = 222
 const EMISSION_ADMIN = 333
+const EMISSION_MANAGER = 444
+const REWARD_1 = 555
+const REWARD_2 = 666
 
 @view
 func __setup__{syscall_ptr : felt*, range_check_ptr}():
     %{
         context.emission_manager = deploy_contract("./contracts/rewards/base/emission_manager.cairo", [ids.OWNER, ids.REWARDS_CONTROLLER]).contract_address
+        context.rewards_controller_1 = deploy_contract("./contracts/rewards/base/rewards_controller.cairo", [ids.EMISSION_MANAGER]).contract_address
+        context.rewards_controller_2 = deploy_contract("./contracts/rewards/base/rewards_controller.cairo", [ids.EMISSION_MANAGER]).contract_address
     %}
     return ()
 end
 
+func get_contract_addresses() -> (
+    emission_manager : felt,
+    rewards_controller_1 : felt,
+    rewards_controller_2 : felt
+):
+    tempvar emission_manager
+    tempvar rewards_controller_1
+    tempvar rewards_controller_2
+    %{ 
+        ids.emission_manager = context.emission_manager
+        ids.rewards_controller_1 = context.rewards_controller_1
+        ids.rewards_controller_2 = context.rewards_controller_2
+    %}
+    return (emission_manager, rewards_controller_1, rewards_controller_2)
+end
+
 @external
 func test_constructor{syscall_ptr : felt*, range_check_ptr}():
+    alloc_locals
+    let (local emission_manager, local __, local ___) = get_contract_addresses()
```

Comment:
a single `_` is enoughn right?

Reply:
I'm not sure why, but this happens:
![image](https://user-images.githubusercontent.com/11295946/178940765-0c831ac5-9f8e-49d4-98d0-625eb53c1fe0.png)

I can use only two for each and overwrite the variable, that would be fine.